infra/src/dsl/bff.ts

```
@@ -2,22 +2,43 @@ import { json, ref } from './dsl'
+ * Trim any leading and trailing slashes
+ */
+const sanitizePath = (path: string) => path.replace(/^\/+|\/+$/g, '')
+
 export const getScopes = (key: PortalKeys) => {
   switch (key) {
     case 'minarsidur':
-      return servicePortalScopes
+      const combinedScopes = new Set([
+        ...servicePortalScopes,
+        ...applicationSystemScopes,
+      ])
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)